splunk_eventgen/default/eventgen.conf

```diff
@@ -1,4 +1,3 @@
-# Copyright (C) 2005-2015 Splunk Inc. All Rights Reserved.
```

Comment:
Why do we remove this copyright line?

Reply:
It is out of data and the copyright info is located in some file but not others. So I just removed it.